src/ImageSharp/Common/Helpers/SimdUtils.cs

```diff
@@ -25,6 +25,25 @@ internal static partial class SimdUtils
         public static bool HasVector8 { get; } =
             Vector.IsHardwareAccelerated && Vector<float>.Count == 8 && Vector<int>.Count == 8;
 
+        /// <summary>
+        /// Gets a value indicating whether <see cref="Vector{T}"/> code is being JIT-ed to SSE instructions
+        /// where float and integer registers are of size 128 byte.
+        /// </summary>
+        public static bool HasVector4 { get; } =
```

Comment:
I find this name misleading, one may think it has to do something with `Vector4`.

Reply:
Any suggestions?

Reply:
I would just inline it, there would be only one usage, if we'd address my other suggestion.